examples/step-13/step-13.cc

```diff
@@ -726,18 +711,16 @@ namespace Step13
     // function may be called multiply, once for each evaluation of the
     // solution which the user required.
     template <int dim>
-    void
-    Solver<dim>::
-    postprocess (const Evaluation::EvaluationBase<dim> &postprocessor) const
+    void Solver<dim>::postprocess(
+      const Evaluation::EvaluationBase<dim> &postprocessor) const
```

Comment:
I very much prefer clang-format's variant of writing these statement. Having `void` and `Solver<dim>::` on separate lines is just awkward. `Solver<dim>::postprocess` should stick together to make it easy to understand what function this is about. Having the parameter on a new line is fine.